topics/statistics/tutorials/fruit_360/workflows/fruit_360.ga

```diff
@@ -0,0 +1,442 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "Image classification with fruit 360 dataset",
+    "format-version": "0.1",
+    "name": "PAG_2022_Fruit_360",
```

Comment:
Would you like to add your authorship/license information here? This can be done from the Galaxy UI  or you can see here for an example https://github.com/galaxyproject/training-material/blob/main/topics/genome-annotation/tutorials/funannotate/workflows/main_workflow.ga#L4

Reply:
> 
Sure, I don't have an identifier. Can I leave that out?

Reply:
yep that's fine!

Reply:
Ok, pushed the change.

Reply:
The first test seems to fail, any clue?

Reply:
it's a dependency issue, it's supposed to be fixed with #3087 but not sure.